src/Traits/Stats.php

```diff
@@ -63,6 +64,19 @@ public function getTotalCharacterMiningIsk(array $character_ids): ?float
             ->total_mined_value;
     }
 
+    /**
+     * @param  int[]  $character_ids
+     * @return float|null
+     */
+    public function getTotalCharacterRattingIsk(array $character_ids): ?float
+    {
+        return CharacterWalletJournal::selectRaw('SUM(amount) as total_ratting_value')
```

Comment:
Any way to avoid `selectRaw()` here?

Reply:
☕ yeah, just late and my copy-pasta game was weak.

Reply:
Updated.